manifests/m/migRaven/GmbH/2.0.0/migRaven.GmbH.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: migRaven.GmbH
+PackageVersion: 2.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /quiet
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OthmanAdi/migRavenNexusWinget/releases/download/v2.0.0/migRaven.Nexus.Bootstrap.exe

```

Comment:
I went looking for the "official" publisher and this is what I found: https://www.aikux.com/en/produkte/migraven/?vid=emn_5depVVbxZZdQBZWj&sid=18AiOscTRl40-1DND_Kl